bin/init_repo.sh

```diff
@@ -4,9 +4,27 @@ nc="\033[0m"
 red="\033[0;31m"
 green="\033[0;32m"
 
+root_path="$(dirname "$0")/.."
+
+printf "Select the PYTHON version: "
+read -r PYTHON_VERSION
```

Comment:
Because it is a bit more that a placeholder: versions other than 3.6,3.7,3.8 and 3.9 are not even allowed. Besides with different versions we also affect which files are picked

Reply:
👍

Reply:
On this note, better specify which ones are indeed allowed?

Reply:
Done!